auv_sim/auv_sim_description/taluy_sim_description/urdf/taluy_sim.urdf.xacro

```diff
@@ -4,7 +4,7 @@
   <xacro:arg name="debug" default="0" />
   <xacro:arg name="namespace" default="taluy" />
   <xacro:arg name="control_rate" default="20" />
-  <xacro:arg name="inertial_reference_frame" default="world" />
+  <xacro:arg name="inertial_reference_frame" default="odom" />
```

Comment:
This is the default argument and let's leave this as is. The spawn launch should be injecting the actual values that we use. So lets modify that. Even in that case should we be changing this to odom ?

Reply:
Oversight on my part, didn't realize it was a default. I am not actually sure about if we should use world or odom, now it seem to didn't have any difference in my testing but we can investigate this further.

reverted this.

Reply:
Right, even though the header frame is "world" all the twist information is still in the body frame. That frame only would effect the position, and for the time being we don't have anıt other pose reference frame. The odom will be published by the ekf node, so no problems as far as the dvl part is concerned.